webpack.common.ts

```diff
@@ -1,77 +0,0 @@
-/*!
```

Comment:
Why are webpack files are removed completely?

Reply:
From what I can tell, the webpack files were being used to build the public website in manuscripts-frontend, which we don't do here.

If I missed something and the files are still needed, let me know.